analyzers/its/SonarAnalyzer.Testing.ImportBefore.targets

```diff
@@ -39,8 +39,8 @@
                       Overwrite="true" />
 
     <ItemGroup>
-      <!-- Remove all previously added analyzers -->
-      <Analyzer Remove="@(Analyzer)" />
+      <!-- Remove all previously added analyzers, except built-in SourceGenerators -->
+      <Analyzer Remove="@(Analyzer)" Condition="!$([System.Text.RegularExpressions.Regex]::IsMatch('%(Filename)', '(Generator|SourceGeneration)$'))"/>
```

Comment:
Original analyzer imported by the SDK (underlined analyzers contain SGs):

![image](https://user-images.githubusercontent.com/103252490/206756260-dd587a26-d141-4bf0-88c7-d741018f43a0.png)

Analyzers passed to csc.exe with this condition:

```
/analyzer:"C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\7.0.0\analyzers/dotnet/cs/Microsoft.Interop.JavaScript.JSImportGenerator.dll" 
/analyzer:"C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\7.0.0\analyzers/dotnet/cs/Microsoft.Interop.LibraryImportGenerator.dll" 
/analyzer:"C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\7.0.0\analyzers/dotnet/cs/Microsoft.Interop.SourceGeneration.dll" 
/analyzer:"C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\7.0.0\analyzers/dotnet/cs/System.Text.Json.SourceGeneration.dll" 
/analyzer:"C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\7.0.0\analyzers/dotnet/cs/System.Text.RegularExpressions.Generator.dll" 
/analyzer:C:\Projects\sonar-dotnet\analyzers\its\..\packaging\binaries\SonarAnalyzer.CFG.dll 
/analyzer:C:\Projects\sonar-dotnet\analyzers\its\..\packaging\binaries\SonarAnalyzer.CSharp.dll 
/analyzer:C:\Projects\sonar-dotnet\analyzers\its\..\packaging\binaries\SonarAnalyzer.dll 
/analyzer:C:\Projects\sonar-dotnet\analyzers\its\..\packaging\binaries\SonarAnalyzer.VisualBasic.dll 
/analyzer:C:\Projects\sonar-dotnet\analyzers\its\..\packaging\binaries\Google.Protobuf.dll
```

Based on https://stackoverflow.com/a/8130142 and https://twitter.com/Nick_Craver/status/1173996405276467202?s=09